ietf-te.yang

```diff
@@ -681,21 +713,42 @@ module ietf-te {
             "Enables the hierarchical link properties supported by
              this tunnel";
         }
+        leaf local-node-id {
+          type nw:node-id;
+          description
+            "The local node identifier.";
+        }
         leaf local-te-node-id {
           type te-types:te-node-id;
           description
             "The local TE node identifier.";
         }
+        leaf local-link-tp-id {
+          type nt:tp-id;
+          description
+            "The local link termination point identifier.";
+        }
         leaf local-te-link-tp-id {
           type te-types:te-tp-id;
           description
             "The local TE link termination point identifier.";
         }
+        leaf remote-link-tp-id {
+          type nt:tp-id;
+          description
+            "The remote link termination point identifier.";
+        }
         leaf remote-te-node-id {
           type te-types:te-node-id;
           description
             "Remote TE node identifier.";
         }
+        leaf network-id {
```

Comment:
We understand the grouping te-toplogy-identifier in both TE tunnel and TE topology.
Can we modify the:
uses te-types:te-topology-identifier and augment an additional leaf network here in TE tunnel?

Reply:
The network-id identifies the the network topology.
The te-topology-identifier container identifies the TE network topology. 
Both can exist independently.